tests/components/bayesian/test_binary_sensor.py

```diff
@@ -66,10 +126,9 @@ def test_sensor_numeric_state(self):
         self.hass.block_till_done()
 
         state = self.hass.states.get("binary_sensor.test_binary")
-        assert [
-            {"prob_false": 0.4, "prob_true": 0.6},
-            {"prob_false": 0.1, "prob_true": 0.9},
-        ] == state.attributes.get("observations")
+        assert state.attributes.get("observations")[0]["prob_given_true"] == 0.6
+        assert state.attributes.get("observations")[1]["prob_given_true"] == 0.9
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5279916)